app/javascript/App/bundles/common/utils/numberFormatters.tsx

```diff
@@ -0,0 +1,67 @@
+const numericSymbols = ['k', 'M', 'B', 'T', 'P', 'E']
+
+export function formatValue(
```

Comment:
Can we add block comments to these helper methods so we know what they output. It looks useful but it's not immediately clear what `value.toLocaleString('en-US', { maximumFractionDigits })` would return. Adding the block comments with just an example output would help as vscode would display it when we hover over the method.